### PR TITLE
Add a pop up dialog box for subscribing to newsletter done ! #306 ✨✨

### PR DESCRIPTION
**Pull request by Subhajit-2023-44**
Hello, PR
Add new feature - #306
Add a pop up dialog box for subscribing to newsletter done properly -

Screenshot 2024-10-26 174107

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] I have followed all the guidlines mentioned in CONTRIBUTING.md.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] I have tested it locally and it works fine.
- [x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [x] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

*Thank you for contributing to Research Nexus! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 done, Sir please merge my PR with level 3, If you check other projects, they are also giving level 3 for this..

Screenshot 2024-10-26 112803

You can check my code..

Thank you 🙏🏻

**Review by Harshdev098 — CHANGES_REQUESTED** [review]
Can you increase the form size to look as same as the image in the left

**Comment by Harshdev098** [comment]
@Subhajit-2023-44 

Screenshot from 2024-10-26 21-11-20

**Comment by Subhajit-2023-44** [comment]
> @Subhajit-2023-44
> 
> Screenshot from 2024-10-26 21-11-20

@Harshdev098 But sir, I have kept the same size of both part -

image

**Comment by Harshdev098** [comment]
Will check it out and merge it by tomorrow

**Comment by Subhajit-2023-44** [comment]
> Will check it out and merge it by tomorrow

sir please check I have kept the same size of both part -


https://github.com/user-attachments/assets/bde5e7ad-f779-43e4-acc6-efdfb9ddcdae

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 sir please merge my PR with level3 otherwise i can't able to another task..

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 sir please merge my PR with level3 otherwise i can't able to another task..

**Comment by Harshdev098** [comment]
@Subhajit-2023-44 I am facing the same issue also after subscribing its not closing the display

https://github.com/user-attachments/assets/5095098e-ebae-4146-a489-9715f4edced6

**Comment by Harshdev098** [comment]
Also the previous issue persists in my firefox, its seems like it worked on chrome

**Comment by Subhajit-2023-44** [comment]
> Also the previous issue persists in my firefox, its seems like it worked on chrome

Ooo, but no problem in Chrome,  I don't use firefox, otherwise I would have checked it there too.  Sir check,  it is 100% zoomed in your browser or maybe it firefox website problem..

@Harshdev098 sir only two days left for the competition gssoc-ext.  Sir please🙏🏻  merge my PR with level3 otherwise i can't able to another task.. Thank You 🙏🏻

File: public/script/popup.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  // });
+
+  // Handle form submission
+  document.getElementById('emailForm-nl').addEventListener('submit', function(event) {
+    event.preventDefault();
+
+    const email = document.getElementById('email-nl').value;
+    if (email) {
+      alert(`Your email ID ${email} has been registered successfully for the newsletter.`);
+      document.getElementById('popup').style.display = 'none';
```
Comment (Harshdev098) [inline public/script/popup.js:26]: Also check it, its not display none once it clicked on subscribe

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

File: public/css/style.css [hunk 2]
```diff
@@ -1875,3 +1875,146 @@ ul {
...
+
+   .popup-right-nl {
+       padding: 20px;
+       width: 50%;
+       display: flex;
+       flex-direction: column; 
+       align-items: center;
+       justify-content: center;
+   }
+
```
[the remark this answers is not in the document]
Comment (Subhajit-2023-44) [reply public/css/style.css:1930]: okay i will try

**Comment by Subhajit-2023-44** [comment]
wait i will give a new PR soon

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 sir done 

Screenshot 2024-10-29 011711

**Pull request closed without merging**